### ISSUE TITLE
Resolve a11y errors flagged in Org Sign Up flow

### ISSUE THREAD

**Issue by georgehager** [CDCgov/prime-simplereport#4582]
## Background

We've added axe accessibility testing to our cypress e2e tests, and have some failures that need to be resolved. Currently, there are some errors flagged in the Organization Sign Up flow, cypress spec 01.

## Action Requested

- Remove the `rules` in the `checkA11y()` calls that allow the axe cypress tests to ignore errors
- Fix the a11y issues

## Acceptance Criteria

Running a Deque audit for the flow and the cypress spec 01 test result in no a11y errors

## Resources & Notes

- The errors can be easily found by searching for `// failing a11y test`
- Note that errors related to the Toast and Address Confirmation Modal are being resolved by #4564 and #4565, those errors do not need to be resolved with this ticket.
